tools/actions/turbo-affected/build/main.js

```diff
@@ -20862,10 +20862,12 @@ var package_default = {
     "coin:near": "pnpm --filter coin-near",
     "coin:polkadot": "pnpm --filter coin-polkadot",
     "coin:solana": "pnpm --filter coin-solana",
+    "coin:icon": "pnpm --filter coin-icon",
+    "coin:stellar": "pnpm --filter coin-stellar",
     "coin:tezos": "pnpm --filter coin-tezos",
     "coin:tron": "pnpm --filter coin-tron",
     "coin:xrp": "pnpm --filter coin-xrp",
-    "coin:ton": "pnpm --filter coin-ton",
```

Comment:
This shouldn't happen I think

Reply:
Damn rebase, good catch! 👍

Reply:
Fixed in this commit https://github.com/LedgerHQ/ledger-live/pull/7750/commits/88a211dc239c6d54eaa61836fe80070f69954265